README.md

```diff
@@ -130,4 +130,22 @@ If you want to use JsonLogic *and* support deprecated browsers, you could easily
 
 It's not possible to include everyone's excellent ideas without the core library bloating, bringing in a ton of outside dependencies, or occasionally causing use case conflicts (some people need to safely execute untrusted rules, some people need to change outside state).
 
+You can perform custom operations on your dataset using `add_operation`
+
+```js
+function lessThanNumber(age) {
+  return age < 65;
+}
+
+function greatherThanNumber(age) {
```

Comment:
This is misleading. `greatherThanNumber` is actually `greaterThanOrEqualToNumber`.